addons/mail/wizard/mail_compose_message.py

```diff
@@ -524,3 +524,7 @@ def render_message(self, cr, uid, wizard, res_id, context=None):
 
     def generate_email_for_composer(self, cr, uid, template_id, res_id, context=None):
         return self.generate_email_for_composer_batch(cr, uid, template_id, [res_id], context)[res_id]
+
+    # Get mention user list (@ Functionality)
+    def get_mention_users(self, cr, uid, word, context=None):
+        return self.pool.get('res.users').search_read(cr, uid, ['|', ('login', 'ilike', word), ('name', 'ilike', word)], ['partner_id', 'login'], limit=8, context=context)
```

Comment:
whats meaning of *word* here? and limit shall be pass from js, where we can configure it.